guide/src/format/configuration/renderers.md

```diff
@@ -150,7 +150,7 @@ The following configuration options are available:
 - **edit-url-template:** Edit url template, when provided shows a
   "Suggest an edit" button (which looks like <i class="fa fa-edit"></i>) for directly jumping to editing the currently
   viewed page. For e.g. GitHub projects set this to
-  `https://github.com/<owner>/<repo>/edit/master/{path}` or for
+  `https://github.com/<owner>/<repo>/edit/main/{path}` or for
   Bitbucket projects set it to
   `https://bitbucket.org/<owner>/<repo>/src/master/{path}?mode=edit`
```

Comment:
Does Bitbucket default to master? It seems strange to be inconsistent here.

Reply:
I just created an empty repo on bitbucket.org and it defaulted to "master" (similar to what I saw in other bitbucket repos).

So, I believe the patch is correct.